Decred Wallet.xcodeproj/project.pbxproj

```diff
@@ -143,6 +139,11 @@
 /* Begin PBXFileReference section */
 		23F5A25CDDC0ADF47F98ED95 /* Pods-Decred Wallet.mainnet release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-Decred Wallet.mainnet release.xcconfig"; path = "Target Support Files/Pods-Decred Wallet/Pods-Decred Wallet.mainnet release.xcconfig"; sourceTree = "<group>"; };
 		2506406D2304A2DE0028A9AC /* Dcrlibwallet.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Dcrlibwallet.framework; path = libs/Dcrlibwallet.framework; sourceTree = "<group>"; };
+		2543694B233E39F200E848AC /* Dcrlibwallet.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Dcrlibwallet.framework; path = ../../../../Downloads/Dcrlibwallet.framework/Dcrlibwallet.framework; sourceTree = "<group>"; };
+		2569B20023436D0E0070D9A2 /* Dcrlibwallet.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Dcrlibwallet.framework; path = "Decred Wallet/libs/Dcrlibwallet.framework"; sourceTree = "<group>"; };
```

Comment:
3 `Dcrlibwallet.framework` listed here. Please correct.

Reply:
resolved in 72e13de